.all-contributorsrc

```diff
@@ -53,7 +53,17 @@
       "contributions": [
         "code"
       ]
+    },
+    {
```

Comment:
don't necessarily need this. all-contributors will take care of it.

Reply:
Since we target the Spring Boot universe, the only version specified is that of the Spring Boot Release (used in the BOM import). Everything else that we don't know better should be taken from there.

Relying on Pivotal seems to be more effective than tuning yourself, since it is related to compile Deos only.

I separated Kotlin out of it, because they release more often than 1-2 times a year.

For the maven plugins, I specified all versions in plugin management...